apidef/api_definitions.go

```diff
@@ -797,9 +797,31 @@ type UpstreamBasicAuth struct {
 	Username string `bson:"username" json:"username"`
 	// Password is the password to be used for upstream basic authentication.
 	Password string `bson:"password" json:"password"`
-	// HeaderName is the custom header name to be used for upstream basic authentication.
+	// Header holds the configuration for custom header name to be used for upstream basic authentication.
```

Comment:
Not sure why this is changed?

Reply:
as part of the contract change we had discussed yesterday - header should have capability to be enabled/disabled and can be extended to other auth sources

Reply:
not noted on ticket